### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by eubyte** [kimhou/replace-bundle-webpack-plugin#1]
Can you please correct the spelling on 'partten' -- it should be 'pattern'. Thank you.

**Comment by zhex** [comment]
yes, meet the same problem , please merge the pull request and release the patch, thx.

**Comment by MikeMcElroy** [comment]
Happy belated birthday, `partten` typo! 🎉 🎈 🎂

**Comment by krmao** [comment]
very good!

**Comment by licaomeng** [comment]
@kimhou Oops, this is an old thread, Kearns' typo correction hasn't been published to NPM, a breaking change release would be needed.
